sites/main-site/src/content/blog/2024/seqera-containers-part-1.mdx

```diff
@@ -0,0 +1,453 @@
+---
+title: 'Migration from Biocontainers to Seqera Containers: Part 1'
+subtitle: What Seqera Containers is and why we want to move to it.
+pubDate: 2024-09-12T09:00:00+01:00
+headerImage: https://images.unsplash.com/photo-1517061391662-b09454c10462
+headerImageAlt: Photo by Barth Bailey on Unsplash
+authors:
+    - 'ewels'
+    - 'edmundmiller'
+label:
+    - 'modules'
+    - 'wave'
+    - 'seqera containers'
+embedHeaderImage: false
+---
+
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+import butwhy from '@assets/images/blog/seqera-containers-part-1/simone-secci-49uySSA678U-unsplash.jpg';
+import mulled from '@assets/images/blog/seqera-containers-part-1/hannah-pemberton-bXi4eg4jyuU-unsplash.jpg';
+import chillies from '@assets/images/blog/seqera-containers-part-1/anne-nygard-3tPkwnNRRkI-unsplash.jpg';
+import wave_logs from '@assets/images/blog/seqera-containers-part-1/logs_screenshot.png';
+import leaves from '@assets/images/blog/seqera-containers-part-1/providence-doucet-mE5MBZX5sko-unsplash.jpg';
+
+import { Image } from 'astro:assets';
+
+## Introduction
+
+Dear nf-core community, the core team would like to inform you about some upcoming changes in how we would like to handle software containers... :package:
+
+Software containers are a fundamental part of modern bioinformatics workflows.
+Nextflow supports multiple container platforms, but the two most commonly used are Docker and Singularity.
+When using containers, all software requirements for a specific Nextflow process are wrapped up into an image and referenced within the pipeline code.
+The process tasks run in isolation in the host environment, and the end user doesn't need to worry about installing software dependencies for the pipeline.
+The software builds are locked in time, making results highly reproducible over many years.
+
+To ensure maximum compatibility across different environments, nf-core modules are typically set up with a conda `environment.yml`, listing software package dependencies from [Bioconda](https://bioconda.github.io/).
```

Comment:
I don't know whether it would be too much to add an example here:
```suggestion
To ensure maximum compatibility across different environments, nf-core modules are typically set up with a conda `environment.yml`, listing software package dependencies from [Bioconda](https://bioconda.github.io/), see an example [here](https://github.com/nf-core/modules/blob/master/modules/nf-core/fastqc/environment.yml).
```